tracing-subscriber/src/filter/level.rs

```diff
@@ -0,0 +1,136 @@
+use std::{cmp::Ordering, fmt, str::FromStr};
+use tracing_core::Level;
+
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub struct LevelFilter(Inner);
```

Comment:
This crate re-implements its own `LevelFilter` type even though `tracing` also provides one, in order to avoid having to depend on `tracing` (rather than `tracing-core`). 

I'm open to adding an optional `tracing` dependency instead (since the filtering layer is feature-flagged), or opening a ticket to move `LevelFilter` from `tracing` to `tracing-core`; reimplementing it here was intended as a temporary solution.